paddle/fluid/framework/details/multi_devices_graph_builder.h

```diff
@@ -27,15 +27,16 @@ class NCCLContextMap;
 namespace framework {
 class Scope;
 namespace details {
+
 class MultiDevSSAGraphBuilder : public SSAGraphBuilder {
  public:
 #ifdef PADDLE_WITH_CUDA
```

Comment:
let's discuss tomorrow. Does tensorflow use this many GOOGLE_CUDA? I feel PADDLE_WITH_CUDA is everywhere...